### PR TITLE
Stat panel: Update out-of-date README

### PR DESCRIPTION
**Pull request by baldm0mma**
**What is this feature?**

Updating the README for the Stat Panel.

**Why do we need this feature?**

The current README is out of date and describing and linking to the component Stat Panel replaces, named Singelstat Panel.

**Who is this feature for?**

All users.

### CODE REVIEW

File: public/app/plugins/panel/stat/README.md [hunk 1]
```diff
@@ -1,9 +1,18 @@
-# Singlestat Panel - Native Plugin
+# Stat panel - Native Plugin
 
-The Singlestat Panel is **included** with Grafana.
+The Stat panel is **included** with Grafana.
 
-The Singlestat Panel allows you to show the one main summary stat of a SINGLE series. It reduces the series into a single number (by looking at the max, min, average, or sum of values in the series). Singlestat also provides thresholds to color the stat or the Panel background. It can also translate the single number into a text value, and show a sparkline summary of the series.
+The Stat panel visualization shows a one large stat value with an optional graph sparkline. You can control the background or value color using thresholds.
```
Comment (codeincarnate) [inline public/app/plugins/panel/stat/README.md:5]: `one large statistic`, `The background or value color can be controlled`

File: public/app/plugins/panel/stat/README.md [hunk 2]
```diff
@@ -1,9 +1,18 @@
...
 
-Read more about it here:
+By default, the Stat panel displays one of the following:
 
-[http://docs.grafana.org/reference/singlestat/](http://docs.grafana.org/reference/singlestat/)
+- Just the value for a single series or field.
```
Comment (codeincarnate) [inline public/app/plugins/panel/stat/README.md:9]: Remove `just`

File: public/app/plugins/panel/stat/README.md [hunk 3]
```diff
@@ -1,9 +1,18 @@
...
+- Both the value and name for multiple series or fields.
```
Comment (codeincarnate) [inline public/app/plugins/panel/stat/README.md:10]: Remove `Both`

File: public/app/plugins/panel/stat/README.md [hunk 4]
```diff
@@ -1,9 +1,18 @@
...
+
+You can use the Text mode to control whether the text is displayed or not.
```
Comment (codeincarnate) [inline public/app/plugins/panel/stat/README.md:12]: `The text mode can be used to control`

File: public/app/plugins/panel/stat/README.md [hunk 5]
```diff
@@ -1,9 +1,18 @@
...
+
+The Stat panel is a replacement for the Singlestat panel, which was deprecated in Grafana v7.0 and removed after Grafana v8.4. Learn more about the now removed Singlestat panel [here](https://grafana.com/docs/grafana/v8.4/visualizations/stat-panel/).
```
Comment (codeincarnate) [inline public/app/plugins/panel/stat/README.md:14]: `removed in Grafana v8.4`

File: public/app/plugins/panel/stat/README.md [hunk 6]
```diff
@@ -1,9 +1,18 @@
...
+
+Read more about Stat panel on our docs page:
```
Comment (codeincarnate) [inline public/app/plugins/panel/stat/README.md:16]: `on the docs page`

**Review by codeincarnate — COMMENTED** [review]
Looks good! I'm nitpicking the hell out of this but let's make sure even the READMEs shine!

File: public/app/plugins/panel/stat/README.md (hunk 1 above)
Comment (baldm0mma) [reply public/app/plugins/panel/stat/README.md:5]: Fixed.

File: public/app/plugins/panel/stat/README.md (hunk 2 above)
Comment (baldm0mma) [reply public/app/plugins/panel/stat/README.md:9]: Fixed.

File: public/app/plugins/panel/stat/README.md (hunk 3 above)
Comment (baldm0mma) [reply public/app/plugins/panel/stat/README.md:10]: Fixed.

File: public/app/plugins/panel/stat/README.md (hunk 5 above)
Comment (baldm0mma) [reply public/app/plugins/panel/stat/README.md:14]: Fixed.

File: public/app/plugins/panel/stat/README.md (hunk 4 above)
Comment (baldm0mma) [reply public/app/plugins/panel/stat/README.md:12]: Fixed.

File: public/app/plugins/panel/stat/README.md (hunk 6 above)
Comment (baldm0mma) [reply public/app/plugins/panel/stat/README.md:16]: Fixed.

**Review by nmarrs — APPROVED** [review]
🔥

**Pull request merged**